.github/workflows/execution-plan-snippet-earthly.yml

```diff
@@ -0,0 +1,246 @@
+name: "Execution Plan"
```

Comment:
Since we are now copying the workflows:
please create a story which takes care of removing the old workflows, and add a reference (Story-ID) somewhere in the code, e.g.
```suggestion
# This workflow will be removed in <storyId>
name: "Execution Plan"
```